packages/backend/tsconfig.json

```diff
@@ -10,7 +10,7 @@
         "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
         "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
         "lib": ["esnext", "dom"],
-         "allowJs": true,                             /* Allow javascript files to be compiled. */
```

Comment:
unrelated change

Reply:
Yep, seems like prettier pick this one up automatically.